internal/http/services/owncloud/ocs/ocs.go

```diff
@@ -32,10 +32,11 @@ func init() {
 
 // Config holds the config options that need to be passed down to all ocs handlers
 type Config struct {
-	Prefix       string           `mapstructure:"prefix"`
-	Config       ConfigData       `mapstructure:"config"`
-	Capabilities CapabilitiesData `mapstructure:"capabilities"`
-	GatewaySvc   string           `mapstructure:"gateway"`
+	Prefix         string           `mapstructure:"prefix"`
+	Config         ConfigData       `mapstructure:"config"`
+	Capabilities   CapabilitiesData `mapstructure:"capabilities"`
+	GatewaySvc     string           `mapstructure:"gateway"`
+	FilesNamespace string           `mapstructure:"files_namespace"`
```

Comment:
@butonic I haven't found another way to expose this path to the service other than this, since storage registries are not going to be part of the gateway, there is no way to use `GetHome`. Ideas / changes are welcomed

Reply:
@refs now I understand your question for the storage registry in the gateway, we can add the GetHome calls to the gateway, indeed, it makes much sense to do it.

Reply:
The GetHome call will just give you the home path for a user, but the client still doesn't get exposed to the storage registry neither storage providers directly.

Let me add that to the CS3APIS